package.json

```diff
@@ -2,7 +2,7 @@
     "name": "azure-deploy",
     "displayName": "Deploy to Azure",
     "description": "Generating CI/CD pipelines to Azure from GitHub and Azure Repo",
-    "version": "1.0.4",
```

Comment:
lets make this change only we are about to ship a newer version of extension. Otherwise no need to update.

Reply:
Sure